src/app/wallet/v2/view.nim

```diff
@@ -3,18 +3,21 @@ import NimQml, chronicles, stint
 
 import
   ../../../status/[status, wallet],
-  views/[accounts, account_list, collectibles]
+  views/[accounts, account_list, collectibles],
+  views/buy_sell_crypto/[service_controller]
 
 QtObject:
   type
     WalletView* = ref object of QAbstractListModel
       status: Status
       accountsView: AccountsView
       collectiblesView: CollectiblesView
+      cryptoServiceController: CryptoServiceController
```

Comment:
Is it part of the refactor to "new" way of doing? Everything else is being name View rather than Controller

Reply:
@alaibe this is definitely not a view. In our project view/model/controller roles are a bit intertwined, so how it should be, in short:
1. Frontend should contain:
- view is qml part
- each view should have its controller (sometimes named view controller)
- each view should have its model (contains data to be displayed)
2. Backend should contain:
- controllers (services)
- providers (data providers)

renaming all that is refactoring what we're not doing, but at least what we can do is stop using wrong naming.

Reply:
While I agree with you, the views are definitely not views. I d always favour consistency. If I browse the code I may be confused by this new concept. Once again purely a matter of opinion but if you are the only one doing Controller and we are doing View that will just not work ;)

Reply:
Hey guys! 
Sorry for disturb but i agreed with @saledjenic bc this "rules" provided by Qt too.

Reply:
@borismelnik sorry if I was unclear, I am not against renaming views into controllers, to be honest, I don't have any preference on what is the name on those. So whatever suits you will work for me. My point is that we are still adding views in other PR and the controller is located in a view folder. So with the current state, this is a confusing change. I'd much rather prefer to have a dedicated PR to refactor rename views into controller if that is what the name you guys want.